src/ESPiLight.cpp

```diff
@@ -78,60 +79,55 @@ void ICACHE_RAM_ATTR ESPiLight::interruptHandler() {
   volatile PulseTrain_t &pulseTrain = _pulseTrains[_actualPulseTrain];
   volatile uint16_t *codes = pulseTrain.pulses;
 
-  if(pulseTrain.length == 0) {
+  if (pulseTrain.length == 0) {
     duration = now - _lastChange;
     /* We first do some filtering (same as pilight BPF) */
-    if(duration > MIN_PULSELENGTH) {
-      if(duration < MAX_PULSELENGTH) {
-	/* All codes are buffered */
-	codes[_nrpulses] = duration;
-	_nrpulses = (_nrpulses+1)%MAXPULSESTREAMLENGTH;
-	/* Let's match footers */
-	if(duration > mingaplen) {
-	  //Serial.print('g');
-	  /* Only match minimal length pulse streams */
-	  if(_nrpulses >= minrawlen && _nrpulses <= maxrawlen) {
-	    //Serial.print(_nrpulses);
-	    //Serial.print('l');
-	    pulseTrain.length = _nrpulses;
-	    _actualPulseTrain = (_actualPulseTrain + 1)%RECEIVER_BUFFER_SIZE;
-	  }
-	  _nrpulses = 0;
-	}
+    if (duration > MIN_PULSELENGTH) {
+      if (duration < MAX_PULSELENGTH) {
+        /* All codes are buffered */
+        codes[_nrpulses] = duration;
+        _nrpulses = (_nrpulses + 1) % MAXPULSESTREAMLENGTH;
+        /* Let's match footers */
+        if (duration > mingaplen) {
+          // Serial.print('g');
+          /* Only match minimal length pulse streams */
+          if (_nrpulses >= minrawlen && _nrpulses <= maxrawlen) {
+            // Serial.print(_nrpulses);
+            // Serial.print('l');
+            pulseTrain.length = _nrpulses;
+            _actualPulseTrain = (_actualPulseTrain + 1) % RECEIVER_BUFFER_SIZE;
+          }
+          _nrpulses = 0;
+        }
       }
       _lastChange = now;
     }
-  }
-  else {
+  } else {
     Serial.print("_!_");
   }
   return;
 }
 
 void ESPiLight::resetReceiver() {
   int i = 0;
-  for(i=0;i<RECEIVER_BUFFER_SIZE;i++) {
+  for (i = 0; i < RECEIVER_BUFFER_SIZE; i++) {
     _pulseTrains[i].length = 0;
   }
   _actualPulseTrain = 0;
   _nrpulses = 0;
 }
 
-void ESPiLight::enableReceiver() {
-  _enabledReceiver = true;
-}
+void ESPiLight::enableReceiver() { _enabledReceiver = true; }
 
-void ESPiLight::disableReceiver() {
-  _enabledReceiver = false;
-}
+void ESPiLight::disableReceiver() { _enabledReceiver = false; }
```

Comment:
Can we avoid this one-line style?

Reply:
You can do a lot, its just fiddly to divert from the default styles: https://clang.llvm.org/docs/ClangFormatStyleOptions.html

I think what you want in particular is:
```
AllowShortFunctionsOnASingleLine: None
```
or
```
AllowShortFunctionsOnASingleLine: Empty
```

Reply:
No, thanks. It is not worth for a project like this.